Description: Map GPT "BIOS Boot Partition" type to MBR type 0xda ("Non-FS data")
 This partition type is used by GRUB2 to install some parts of GRUB2.
Bug-Debian: http://bugs.debian.org/545190
Author: Julien BLACHE <jblache@debian.org>

Index: refit-0.14/gptsync/lib.c
===================================================================
--- refit-0.14.orig/gptsync/lib.c	2010-04-19 18:37:22.933931755 +0200
+++ refit-0.14/gptsync/lib.c	2010-04-19 18:37:19.182590539 +0200
@@ -83,6 +83,7 @@
     { 0xaf, STR("Mac OS X HFS+") },
     { 0xbe, STR("Solaris Boot") },
     { 0xbf, STR("Solaris") },
+    { 0xda, STR("Non-FS data") },
     { 0xeb, STR("BeOS") },
     { 0xee, STR("EFI Protective") },
     { 0xef, STR("EFI System (FAT)") },
@@ -109,7 +110,7 @@
     // From Wikipedia
     { "\x39\x33\xA6\x8D\x07\x00\xC0\x60\xC4\x36\x08\x3A\xC8\x23\x09\x08", 0x00, STR("Linux Reserved"), GPT_KIND_SYSTEM },
     // From grub2 repository, grub/include/grub/gpt_partition.h
-    { "\x48\x61\x68\x21\x49\x64\x6F\x6E\x74\x4E\x65\x65\x64\x45\x46\x49", 0x00, STR("GRUB2 BIOS Boot"), GPT_KIND_SYSTEM },
+    { "\x48\x61\x68\x21\x49\x64\x6F\x6E\x74\x4E\x65\x65\x64\x45\x46\x49", 0xda, STR("BIOS Boot Partition"), GPT_KIND_SYSTEM },
     // From FreeBSD repository, sys/sys/gpt.h
     { "\xB4\x7C\x6E\x51\xCF\x6E\xD6\x11\x8F\xF8\x00\x02\x2D\x09\x71\x2B", 0xa5, STR("FreeBSD Data"), GPT_KIND_DATA },
     { "\xB5\x7C\x6E\x51\xCF\x6E\xD6\x11\x8F\xF8\x00\x02\x2D\x09\x71\x2B", 0x00, STR("FreeBSD Swap"), GPT_KIND_SYSTEM },
